OpenSim/Moco/MocoGoal/MocoContactImpulseTrackingGoal.h

```diff
@@ -73,47 +74,36 @@ namespace OpenSim {
 
 
     /**
-    \section MocoImpulseTrackingGoal
+    \section MocoContactImpulseTrackingGoal
     Minimize the error between compliant contact force impulse from the model and
-    experimentally measured contact impulse.
-    This class handles multiple groups of contact forces and a single
-    experimental external loads file. Tracking ground reaction forces for the
-    left and right feet in gait requires only one instance of this goal.
+    experimentally measured contact impulse - for a single axis.
+    This class handles the contact impulses from a single contact group and a 
+    single experimental external loads file. Tracking ground reaction impulses for
+    the left and right feet in gait requires separate instances of this goal.
+    Tracking ground reaction impulses for multiple axes requires separate 
+    instances of this goal.
```

Comment:
Ah, I didn't see this before I made my comment above, please feel free to ignore it.